src/log.ts

```diff
@@ -296,23 +278,14 @@ class Log implements LogSeverityFunctions {
   delete(gaxOptions?: CallOptions): Promise<ApiResponse>;
   delete(gaxOptions: CallOptions, callback: DeleteCallback): void;
   delete(callback: DeleteCallback): void;
-  delete(optionsOrCallback?: CallOptions|DeleteCallback, cb?: DeleteCallback):
-      void|Promise<ApiResponse> {
-    const gaxOptions =
-        typeof optionsOrCallback === 'object' ? optionsOrCallback : {};
-    const callback =
-        typeof optionsOrCallback === 'function' ? optionsOrCallback : cb;
+  async delete(gaxOptions?: CallOptions|DeleteCallback): Promise<ApiResponse> {
+    this.logging.projectId = await this.logging.auth.getProjectId();
```

Comment:
Same suggestion as [#452](https://github.com/googleapis/nodejs-logging/pull/452#discussion_r278160990) about just using a local variable instead of updating the instance property.

Reply:
Done